### ISSUE TITLE
Seam generation: add "Sharpest" corner option

### ISSUE THREAD

**Issue by smokeandlights** [bambulab/BambuStudio#2340]
It would be really nice if the z seam generation could follow the sharpest corner of an object. This is a great way to hide seams, and it is laborious and messy using the seam painting tools.

**Comment by QingZhangBambu** [comment]
i will add it to our feature pool and concerned staff will evaluate it

**Comment by NVNDO** [comment]
I totally agree. Many objects would hugely benefit of this and it's a basic feature in all slicers. 

Existing Seam-options are not ideal. See here:

![image](https://github.com/bambulab/BambuStudio/assets/73851760/0cc0c848-d100-4f6b-89e9-c443c1ab6488)

![image](https://github.com/bambulab/BambuStudio/assets/73851760/fa29ea34-6197-47fd-97ac-ad34adb5fd7e)

Being able to paint a seam is nice (but can still look awkward).
See here: 

![image](https://github.com/bambulab/BambuStudio/assets/73851760/d1de5a0a-25aa-44b2-a428-3652b5624d01)

![image](https://github.com/bambulab/BambuStudio/assets/73851760/c938ed33-fe79-465e-b9e9-8d84dc715816)

Seam painting should not be the go to option, rather an alternative if nothing else works.

Pls implement.

**Comment by NVNDO** [comment]
Sitting here since half an hour trying my best, but nothing. 

![image](https://github.com/bambulab/BambuStudio/assets/73851760/494494a8-26a3-4c01-8818-e55383b48922)

Even using the seam-tool, the results are very bad! 

![image](https://github.com/bambulab/BambuStudio/assets/73851760/9a802ced-f9db-44f1-b577-fce1e7f44840)

@QingZhangBambu Please implement. 
It's a standard feature everywhere else.

**Comment by NVNDO** [comment]
I'm done with this!! 

I changed to Orca Slicer. 
There at least back is = back. as not even that functions works properly in Bambu Studio...

![Screenshot (472)](https://github.com/bambulab/BambuStudio/assets/73851760/c1f4f4af-16c6-4e30-9543-ec5892af943d)
